tests/fixups/TraceFixup/Logging.h

```diff
@@ -142,11 +142,15 @@ inline void LogString(const char* name, const wchar_t* value)
 }
 inline std::string InterpretStringA(const char* value)
 {
-    return value;
+    if (value != NULL)  

```

Comment:
Please add {} even if the statement spans one line.

Reply:
Addressed in Commit 1bdc01a3

Reply:
Thank you.